### DISCUSSION TITLE
Plex Watchlist in Kodi

### DISCUSSION THREAD

**Discussion by liquid-ion** [croneter/PlexKodiConnect#2045] [Q&A]
I read through this article, it was very helpful in learning how to customize Kodi to reflect Plex libraries:

https://github.com/croneter/PlexKodiConnect/wiki/Video-Nodes-and-Skins

I remember reading a similar article on adding the Plex watchlist into Kodi, but I can't find it anymore (searched everywhere last night).

Am I dreaming this or could someone share a link / guide on how that is possible? I would like to be able to pull the Plex Watchlist, and if possible add/remove items from it as well.

**Comment by croneter** [comment]
Just implemented in PKC beta version 3.9.1

**Reply by cherkes070** [reply]
Is adding / removing from watchlist planned to be implemented?
